fastlane/fastfiles/management-ui.rb

```diff
@@ -11,7 +11,7 @@
                 ":mockzilla-management-ui:android:bundleDebug"
             ]
         )
-        update_reference_screenshots
+        # update_reference_screenshots
```

Comment:
Was this intentionally left it?

Reply:
Yeah that's intentional, the screenshots aren't really doing much at the moment except slowing everything down since there's so much churn in the UI. (Also they're broken and it's not the best use of time to fix them at the moment)

Reply:
Ah okay, wasn't aware if they had been fixed 👍